go/libkb/env.go

```diff
@@ -553,10 +553,16 @@ func (e *Env) GetRootConfigFilename() (string, error) {
 	return filepath.Join(dir, "config.json"), nil
 }
 
+func (e *Env) GetEnvFileDir() string {
+	// Do not respect $XDG_CONFIG_HOME due to debian systemd 229 not supporting %E
```

Comment:
It's confusing to have a comment about XDG here in what appears to be a general-use function.  Can you name it `GetNixEnvFileDir` or something?  Or move it into the platform-specific files and panic on non-nix systems?

Reply:
good point, i made it return an error if in a unsupported runtimegroup like the other functions.